repair-service/dao-api/src/test/java/com/epam/repair/dao/CityDaoTest.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.repair.dao;
+
+import com.epam.repair.model.City;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = Config.class)
```

Comment:
It would be great to name it `TestConfig` to clarify that it's just for test.
Someone can import real config instead of test

Reply:
Comment accepted. Config.class --> TestConfig.class.